Extension/package.json

```diff
@@ -42,6 +42,168 @@
   ],
   "main": "./dist/main",
   "contributes": {
+    "walkthroughs": [
+      {
+        "id": "cppWelcome",
+        "title": "Get Started with C++ Development",
```

Comment:
Following the pattern VS Code has with their other walkthroughs, it should be "Get" (for example, they have "Learn the fundamentals" instead of "Learning the fundamentals" and "Customize your setup" instead of "Customizing")

Reply:
Ok